packages/react-components/react-badge/library/src/components/Badge/useBadgeStyles.styles.ts

```diff
@@ -174,32 +180,32 @@ const useRootStyles = makeStyles({
     ...shorthands.borderColor('currentColor'),
   },
   'outline-brand': {
-    color: tokens.colorBrandForeground1,
+    color: semanticTokens.statusBrandTintForeground,
   },
   'outline-danger': {
-    color: tokens.colorPaletteRedForeground3,
-    ...shorthands.borderColor(tokens.colorPaletteRedBorder2),
+    color: semanticTokens.statusDangerTintForeground,
+    ...shorthands.borderColor(semanticTokens.statusDangerStroke),
   },
   'outline-important': {
-    color: tokens.colorNeutralForeground3,
-    ...shorthands.borderColor(tokens.colorNeutralStrokeAccessible),
+    color: semanticTokens.statusImportantTintForeground,
+    ...shorthands.borderColor(semanticTokens.statusImportantStroke),
   },
   'outline-informative': {
-    color: tokens.colorNeutralForeground3,
-    ...shorthands.borderColor(tokens.colorNeutralStroke2),
+    color: semanticTokens.statusInformativeTintForeground,
+    ...shorthands.borderColor(semanticTokens.statusInformativeStroke),
   },
   'outline-severe': {
-    color: tokens.colorPaletteDarkOrangeForeground3,
+    color: tokens.colorPaletteDarkOrangeForeground3,//come back to this one
   },
   'outline-subtle': {
-    color: tokens.colorNeutralForegroundStaticInverted,
+    color: tokens.colorNeutralForegroundStaticInverted,//come back to this one
```

Comment:
Having comments for these is great, let's just add some context here that no semantic token equivalent currently exists and is why they were left for now